docs/reference/triggers/http.md

```diff
@@ -9,13 +9,13 @@ The HTTP trigger is the only trigger created by default if not configured (by de
 | port | int | The NodePort (or equivalent) on which the function will serve HTTP requests. If empty, chooses a random port within the platform range. |
 | ingresses.(name).host | string | The host to which the ingress maps. |
 | ingresses.(name).paths | list of strings | The paths that the ingress handles. Variables of the form `{{.<NAME>}}` can be specified using `.Name`, `.Namespace`, and `.Version`. For example, `/{{.Namespace}}-{{.Name}}/{{.Version}}` will result in a default ingress of `/namespace-name/version`. |
-| readBufferSize | int | Per-connection buffer size for requests' reading. |
-| cors.enabled | boolean | Whether cors is enabled per function. |
-| cors.allowOrigin | string | Indicates where the CORS originates from (Default: '*'). |
-| cors.allowMethods | list of strings | Indicates which methods can be used during the actual request (Default: HEAD, GET, POST, PUT, DELETE, OPTIONS). |
-| cors.allowHeaders | list of strings | Indicates which header field names can be used during the actual request (Default: Accept, Content-Length, Content-Type, X-nuclio-log-level). |
-| cors.allowCredentials | boolean | Indicates that the actual request can include user credentials (Default: false). |
-| cors.preflightMaxAgeSeconds | int | Indicates how long the results of a preflight request can be cached in a preflight result cache (Default: -1, not cached) |
+| readBufferSize | int | Per-connection buffer size for reading requests. |
+| cors.enabled | bool | `true` to enable cross-origin resource sharing (CORS) for each function; (default: `false`). |
```

Comment:
```suggestion
| cors.enabled | bool | `true` to enable cross-origin resource sharing (CORS) for the function; (default: `false`). |
```
function <-> trigger  is 1:n so a trigger configuration refers to one function (multiple functions are not relevant in this context)

Reply:
I already removed "for each function" per Liran's suggestion — see the review comments and commit c4e882e.